client/app/views/trackView.js

```diff
@@ -104,14 +104,25 @@ define(['jquery', 'controllers/analystController', 'Ladda', 'bootstrap'], functi
     }
 
 
+
+
+
+
     // Generate new participation links
     $('#participants-submit').on('click', function (e) {
       e.preventDefault();
 
+      var pc = parseInt($('#participants-count').val());
+      
+      if (pc < 1.0 || pc > 100.00) {
```

Comment:
Can we just use the number type with min and max? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number

Reply:
Can't believe I'm linking to W3Schools, but ... https://www.w3schools.com/html/tryit.asp?filename=tryhtml_input_number

The user can input a higher number, but when trying to submit the browser should stop them. Of course this is only a client-side check, and browser-dependent, so if we care enough we should implement the same check on the server.

Reply:
I think we have already discussed this but the code above is doing the client-side check.
Otherwise just setting max/min doesn't execute a check.